app/components/PrintTable/index.scss

```diff
@@ -23,13 +23,13 @@
 
   &__row {
     font-size: 10pt;
-    border-bottom: 1px solid $palette__text--hint;
-    page-break-inside: avoid;
+    border-bottom: 0.5px solid lightgray;
 
     & td {
-      padding: 10px 4px;
+      padding: 3px 4px;
       max-width: 300px;
       overflow-wrap: break-word;
+      break-inside: avoid;
```

Comment:
what prompted the change from row into td for break?

Reply:
I believe this was to avoid the double border issue when the tables had to break between pages. ClinInfo wants everything to be condensed with no more page breaks between the tumour summary, TO tables, and key alterations so it's likely that the tables with more info will be split between pages